src/model/color.js

```diff
@@ -39,11 +32,71 @@ export default class Color {
                     alpha: zeplinColor.a
                 });
             }
+            case "hsla": {
+                return new Color({
+                    name: zeplinColor.name,
+                    hsla: this.rgbaToHsla(
+                        zeplinColor.r,
+                        zeplinColor.g,
+                        zeplinColor.b,
+                        zeplinColor.a
+                    )
+                });
+            }
             default:
         }
     }
 
     static fromZeplinColors(zeplinColors, context) {
-        return zeplinColors.map(zeplinColor => Color.fromZeplinColor(zeplinColor, context));
+        return zeplinColors.map(zeplinColor =>
+            Color.fromZeplinColor(zeplinColor, context)
+        );
```

Comment:
Is there a specific reason why you changed the formatting? If not, could you revert this to keep things consistent?

Reply:
I’ll revert, I guess my formatter messed up